Content.Shared/Mobs/Components/MobThresholdsComponent.cs

```diff
@@ -18,6 +19,25 @@ public sealed partial class MobThresholdsComponent : Component
     [DataField("currentThresholdState")]
     public MobState CurrentThresholdState;
 
+    /// <summary>
+    /// The health alert that should be displayed for player controlled entities.
+    /// Used for alternate health alerts (silicons, for example)
+    /// </summary>
+    [DataField("healthAlert")]
+    public AlertType HealthAlert = AlertType.HumanHealth;
+
+    [DataField("critAlert")]
+    public AlertType CritAlert = AlertType.HumanCrit;
+
+    [DataField("deadAlert")]
+    public AlertType DeadAlert = AlertType.HumanDead;
+
```

Comment:
This should just be Dictionary<MobState, Alert> to be significantly more flexible.